content/cpp/concepts/pointers/pointers.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: "Pointers"
+Subjects:
+  - "Computer Science"
+  - "Game Development"
+Tags: 
+  - "Pointers"
+CatalogContent:
+  - "learn-c-plus-plus"
+  - "paths/computer-science"
+---
+
+A pointer is a speacial type variable that stores address of another variable in it. 
+
+## Declaration and Initialization of Pointer variable
+
+A pointer variable points to the same data type and is created with the `*` operator. Then the address of another variable is stored in it.
+
+```cpp
```

Comment:
This might be something to do with my machine, but I had to prepend `std::` to `string` and `cout`. Is this something that should be pointed out for our readers?

Reply:
Most of the people declare it after the required header files example: using namespace std;
You want me to explicitly mention std:: with every cout and cin ?

Reply:
Yeah, unfortunately, that's the way we decided to do this for Codecademy lessons. Bjarne Stroustrup also specifically has told me to do so as well when we collab'ed on it.

Reply:
@sonnynomnom  ok, I will explicitly mention it in the entry.